### ISSUE TITLE
Ability for plugins to generate Explore URLs

### ISSUE THREAD

**Issue by legander** [grafana/grafana#60097]
**What would you like to be added:**

I would like to, from an _app plugin_ be able to generate an Explore link given a panelModel. 
There is this handy function called [getExploreURL](https://github.com/grafana/grafana/blob/main/public/app/core/utils/explore.ts) but it's unfortunately not accessible by plugin code.

**Why is this needed:**
We are using the PanelRenderer in an app plugin to visualize timeseries data and would like to expose a button for users to "open this chart in Explore", where they would be able to dig deeper and do further analysis. Changing aggregation, editing tags and pull in additional metrics to correlate with.

![Image](https://user-images.githubusercontent.com/1597337/206725029-d642bb5c-e53e-4c66-b3f8-b7670e7aafd0.png)

cc @Elfo404

**Comment by gelicia** [comment]
Build an exposed version of  `locationUtil.assureBaseUrl(`/explore?left=${encodeURIComponent(serializeStateToUrlParam(exploreUrlState))}`)` in react and go

**Comment by joeblubaugh** [comment]
This would be a really useful feature for me. I would vote this as "not stale."

**Comment by Elfo404** [comment]
yep, def not stale. We also think this would be really useful. thanks for the ping!

**Comment by ifrost** [comment]
This would also help simplify adding "go to Explore" button, like in https://github.com/grafana/explore-logs/blob/f51e29c2fb3ee5fe4198ef0a69efb666370b9ed7/src/Components/ServiceScene/GoToExploreButton.tsx#L16
